README.md

```diff
@@ -84,9 +84,17 @@ The format for listing your product `JSON` includes three sections: 1) required,
     "icd_503": "add the value "true" or url to certification",
     "hipaa": "add the value "true" or url to compliance doc",
     "pci": "add the value "true" or url to compliance doc"
-    
+
 ### How to run this locally
 
+Clone this repo https://github.com/mbland/jekyll_pages_api_search/ and update your `Gemfile` to set the url path to this gem in your local directory
+
+Within the apps-gov repo, run `./go init` which installs the npm modules (specified in package.json). Specifically, we need browserify and uglifyify to compile the custom js/products.js code into js/products-bundle.js, as specified in the jekyll_pages_api_search: browserify: property of _config.yml.
+
+Then, run `./go build` and `./go serve` to run the `bundler-aware` Jekyll build and serve commands, respectively. The .go script also sets the `NODE_PATH environment` variable to add the `node_modules` directory, so that the locally-installed browserify and uglifyify modules are discoverable.
+
+Navigate to
+
```

Comment:
I believe this section can be removed now since it has been updated by @mbland with more detailed instructions